app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -358,19 +359,81 @@ class TopicFragmentTest {
     }
   }
 
+  @Test
+  fun testTopicFragment_withStoryId_clickOnPracticeTab_configChange_showsSameTabAndItsContent() {
+    launchTopicPlayStoryActivityIntent(
+      internalProfileId,
+      FRACTIONS_TOPIC_ID,
+      FRACTIONS_STORY_ID_0
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+      clickTabAtPosition(position = PRACTICE_TAB_POSITION)
+      testCoroutineDispatchers.runCurrent()
+      matchStringOnListItem(
+        recyclerView = R.id.topic_practice_skill_list,
+        itemPosition = 0,
+        targetViewId = R.id.master_skills_text_view,
+        stringToMatch = "Master These Skills"
+      )
+      onView(isRoot()).perform(orientationLandscape())
```

Comment:
could you please confirm one more thing?
When running this test on espresso, is it changing the orientation or it's just passing?

Reply:
If it is not changing orientation try running `testTopicFragment_clickOnReviewTab_configChange_showsSameTabAndItsContent` and see if this is changing the orientation. 

I think this is possible because of time delays, but let's just confirm to prevent any other cause.

Reply:
Yes this test is changing the orientation and then passing

Reply:
@FareesHussain Could you please this behaviour?

Reply:
@anandwana001 @MaskedCarrot Sorry for the delay. I've tested this and I was unable to see the orientation change.
```suggestion
      testCoroutineDispatchers.runCurrent()
      onView(isRoot()).perform(orientationLandscape())
```
Using this I was able to see the orientation change

Reply:
Thanks, @FareesHussain for confirming. 
@MaskedCarrot Please check the above comment.

Reply:
I checked and found that  I was able to see the orientation change on my real device but i did not see it when I ran it on an emulator 
Thanks @anandwana001 @FareesHussain

Reply:
This is a good one as you are able to run the test on a real device. 
Could you tell is there any specific configuration you had to do before running espresso test on a real device?

Reply:
no, I didn't do any changes to my device for these tests in particular. Though to enable my device to run espresso tests I had to disable battery optimization for oppia app.